src/NHSD.BuyingCatalogue.Identity.UserDatabase/Deployment/CreateTestUsers.sql

```diff
@@ -1,28 +1,35 @@
 ﻿DECLARE @aliceEmail AS nvarchar(50) = N'AliceSmith@email.com';
 DECLARE @bobEmail AS nvarchar(50) = N'BobSmith@email.com';
+DECLARE @sueEmail AS nvarchar(50) = N'SueSmith@email.com';
 
 IF '$(INSERT_TEST_DATA)' = 'True'
 AND NOT EXISTS (
   SELECT *
   FROM dbo.AspNetUsers
-  WHERE UserName IN (@aliceEmail, @bobEmail))
+  WHERE UserName IN (@aliceEmail, @bobEmail, @sueEmail))
 BEGIN
     DECLARE @ccgRoleId AS nchar(4) = 'RO98';
     DECLARE @executiveAgencyRoleId AS nchar(5) = 'RO116';
+    DECLARE @HullCCGOdsCode AS nchar(3) = '03F';
```

Comment:
Very minor but all the other variables start with a lowercase character, for consistency reasons it would be ideal if it could match.